README.md

```diff
@@ -1,24 +1,158 @@
+- [x] Understand pros and cons of different approaches of connecting Ruby on Rails back-end with React front-end.
+- [x] Understand principles of Ruby on Rails and React frameworks.
+- [x] Apply Ruby best practices and language style guides in code.
+- [x] Apply RoR best practices and language style guides in code (e.g. thin controllers).
+- [x] Apply JavaScript best practices and language style guides in code.
+- [x] Apply React best practices and language style guides in code.
+- [x] Learn about and practice giving constructive feedback to teammates.
+- [x] Perform a code review for a team member.
+- [x] Use the "Review change"" feature from GitHub.
+- [x] Write clear, professional, and respectful review comments for other team members.
+- [x] Explain "why" a change is requested when giving a code review.
+- [x] Plan a 2+ week project with no interim Microverse-set milestones and submit it on time.
+- [x] Apply knowledge of setting working agreements to set group project teams up for success.
+- [x] Independently implement best practices for group agreements to improve teamwork in larger group project.
+- [x] Demonstrate ability to apply best practices of communication for resolving teamwork challenges.
+- [x] Understand that respect is the foundation of strong relationship-building with teammates.
+- [x] Show up throughout group projects as a reliable and committed team member who communicates and manages your time effectively.
+- [x] Recognize the importance of investing in good working relationships with teammates.
+- [x] Understand principles of strong teamwork (being reliable, committed, and consistent) and how to apply them in group projects.
+- [x] Recognize the value of making equal contributions to group projects to produce the best outcome.
+- [x] Use empty Kanban board to manage tasks with team and own time on the project.
 
-* Database initialization
 
-* How to run the test suite
 
-* Services (job queues, cache servers, search engines, etc.)
+## Project Documentation 📄
 
-* Deployment instructions
+- [x] Here is the presentation video [link](https://drive.google.com/file/d/1Fi1Og-ZkgPK-UnpXF65DkXFkomawnbAF/view?usp=sharing)👈
 
-* ...
+## Built With 🛠️
+
+- [x] Programming Language: [Ruby](https://www.ruby-lang.org/en/)
+- [x] Linter: [Rubocop](https://rubocop.org/)
+- [x] VS Code
+
+## Getting Started
+
+To get a local copy up and running follow these simple steps.
+
+### Prerequisites
+
+- [x] A web browser like Google Chrome.
+- [x] A code editor like Visual Studio Code with Git and Ruby.
+
+You can check if Git is installed by running the following command in the terminal.
+
+```
+$ git --version
+```
+
+Likewise for Ruby installation.
+
+```
+$ ruby --version && irb
+```
+
+### Setup
+
+Clone the repository using the GitHub link provided below.
+
+### Install
+
+In the terminal, go to your file directory and run this command.
+
+```
+$ git clone git@github.com:ger619/ruby-capstone.git
```

Comment:
- [ ] This link does not point to the project's repo.  It points to your Ruby Capstone project.  Please, kindly update this link so it points to the correct project. 💪